components/compact-log-backup/src/test_util.rs

```diff
@@ -466,9 +466,32 @@ impl TmpStorage {
 
     #[track_caller]
     pub fn verify_result(&self, res: SubcompactionResult, mut cm: CompactInMem) {
-        let sst_path = self.path().join(&res.meta.sst_outputs[0].name);
-        res.verify_checksum().unwrap();
-        verify_the_same::<RocksEngine>(sst_path, cm.must_iter()).unwrap();
+        // Verify SST file name format
+        for file in &res.meta.sst_outputs {
+            let sst_path = self.path().join(file.get_name());
+            let file_name = sst_path
+                .file_name()
+                .expect("SST path should have a file name")
+                .to_str()
```

Comment:
why not direct `file.get_name()`?

Reply:
because file.get_name() will `compaction_xxx/outputs/xxx_xx_x_x_xx.sst`